pkg/apiserver/event/collect/system_info_collect.go

```diff
@@ -322,7 +322,17 @@ func genClusterCountInfo(num int) string {
 		return "<10"
 	case num < 50:
 		return "<50"
+	case num < 100:
+		return "<100"
+	case num < 125:
+		return "<125"
+	case num < 150:
+		return "<150"
+	case num < 200:
+		return "<200"
+	case num < 500:
+		return "<500"
 	default:
-		return ">=50"
+		return ">=500"
```

Comment:
Maybe this part of logic could be made more intelligent? Although lots of "case" sentences work.

Reply:
Yes, will find a more flexible algorithm in the future.